c2corg_api/views/waypoint.py

```diff
@@ -28,7 +29,11 @@
 from functools import lru_cache
 from sqlalchemy.orm import joinedload, load_only
 from sqlalchemy.orm.util import aliased
-from sqlalchemy.sql.expression import and_
+from sqlalchemy.sql.elements import literal_column
+from sqlalchemy.sql.expression import and_, union
+
+# the number of routes that are included for waypoints
+NUM_ROUTES = 30
```

Comment:
> There are several such examples in v5 :)
> We'll have to explain the client why this can't be done in v6.

And these pages take 30 seconds to load, they are very unpractical because you are not able to find a route in a list of 100+ entries and they would take a ridiculous amount of space with the card layout. For me it is not an option to show 100+ routes on one page.

> You don't answer my question about how to choose the route to eventually display in the WP page.
> I think this is not trivial.

I don't think it is important which routes you show if you have 30+ routes. If you have more than 30 you click on the link and then you can better browse them.